app/browser/select-dataset-step.tsx

```diff
@@ -195,7 +198,13 @@ const SelectDatasetStepContent = ({
     leading: true,
   });
   const router = useRouter();
-  const classes = useStyles({ datasetPresent: !!dataset, variant });
+  const isOdsIframe = router.query["odsiframe"] === "true";
```

Comment:
@noahonyejese could be good to extract to an util function, as it's now used across two different files 👍